Classes/Player/AVPlayerEngine/AVPlayerEngine+Observation.swift

```diff
@@ -230,6 +233,31 @@ extension AVPlayerEngine {
         }
     }
     
+    /// Handle changes in Audio/Video tracks codecs.
+    private func handleTracksCodecs() {
+        if #available(iOS 13.0, tvOS 13.0, *) {
+            
+            if let playerItem = currentItem {
+                playerItem.tracks.forEach { track in
+                    
+                    if let videoFormatDescriptions = track.assetTrack?.formatDescriptions as? [CMFormatDescription] {
+                        
+                        videoFormatDescriptions.forEach { formatDescription in
+                            switch formatDescription.mediaType {
+                            case .video:
+                                self.post(event: PlayerEvent.VideoTrackChanged(codecDescription: formatDescription.mediaSubType.description))
```

Comment:
@chausov I think we have to fire it only if it has changed i am concerned we will get too many events which app does not need

Reply:
We can not store last value in MessageBus, you have to store it on frontend client or somewhere else outside of the Player.